### PR TITLE
Upgrade to vue 2.0

### PR DESCRIPTION
**Pull request by 52admln**
Upgrade this component to Vue 2.0 ！^_^

### CODE REVIEW

**Comment by Yuyz0112** [comment]
It looks good to me.
Shall we add some more comprehensive description of this PR. Like how did these webpack config changes, maybe related to the new vue-cli template?

File: src/main.js [hunk 1]
```diff
@@ -1,8 +1,13 @@
+// The Vue build version to load with the `import` command
+// (runtime-only or standalone) has been set in webpack.base.conf with an alias.
 import Vue from 'vue'
 import App from './App'
 
+Vue.config.productionTip = false
+
 /* eslint-disable no-new */
 new Vue({
-  el: 'body',
```
Comment (52admln) [inline src/main.js:6]: It need a root element for Vue.js 2.0

**Comment by 52admln** [comment]
Yeah, I changed those files (`build folder and config folder`) related to the new vue-cli template, and `Notification.vue、main.js、index.html` files changed related to the Upgrade Guide, see [Migration from Vue 1.x](https://vuejs.org/v2/guide/migration.html).

**Comment by Yuyz0112** [comment]
@52admln 
I have just read vue's 2.0 doc and I found this: https://vuejs.org/v2/guide/components.html#sync-Modifier
It seems `.sync` is slightly different in 2.0 and it will **only worked in version 2.3+**, just the version you updated to.

Since vue-notie is more like a guide project than a production ready library, I think use the latest version of vue is fine.
But to avoid some misunderstanding on your 'update to vue 2.0' commit, would you mind rename the commit and add this to the top of `README.md`?

**Pull request merged**